.gitlab-ci.yml

```diff
@@ -1,16 +1,19 @@
 image: node:12.18.3
 
 stages:
-  - build
-  - test
+  - tests
 
-build:
-  stage: build
-  script:
-    - node -v
-    - npm install
+cache:
+  paths:
+    - node_modules/
 
-test:
-  stage: test
+tests:
```

Comment:
Why call the stage tests? While you are running multiple test scripts, in my view the stage is still test. This is not gating the approval, just an observation. Feel free to change or not.

Reply:
When I was looking into how to structure the yml files for gitlab, they would always declare their stages at the top, then in the scripts, declare what stage they're in. I'm basically just following convention even though I only have 1 stage.

Example: https://gitlab.com/gitlab-org/gitlab-foss/-/issues/66934

Reply:
Unless you meant 'Test' instead of 'Tests'. I'll make that change before the merge if you approve of the changes.

Reply:
Yes, I was highlighting the plurals tests vs test.